### PR TITLE
Fix multi-language spellcheck

### PR DESCRIPTION
**Pull request by Fatalll**

### CODE REVIEW

File: src/main/kotlin/tanvd/grazi/language/LangTool.kt [hunk 1]
```diff
@@ -20,7 +20,9 @@ object LangTool : GraziLifecycle {
                 lang.configure(this)
                 disableRules(allActiveRules.map { it.id }.filter { it in GraziConfig.state.userDisabledRules })
                 //Spellcheck will be done by Grazi spellchecker
-                disableRules(allActiveRules.filter { it.isDictionaryBasedSpellingRule }.map { it.id })
+                if (arrayOf(Lang.AMERICAN_ENGLISH, Lang.BRITISH_ENGLISH).any { it == lang }) {
```
Comment (TanVD) [inline src/main/kotlin/tanvd/grazi/language/LangTool.kt]: You should check if this is is English at all, not if it is one of a dialects

File: src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import tanvd.grazi.GraziTestBase
+import tanvd.grazi.ide.GraziLifecycle
+import tanvd.grazi.language.Lang
+
+
+class MultiLanguageTextSupportTest : GraziTestBase(true) {
+    fun `test grammar check in file`() {
+        val isRussianEnabled = GraziConfig.state.enabledLanguages.contains(Lang.RUSSIAN)
+
+        if(!isRussianEnabled) {
```
Comment (TanVD) [inline src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt]: Use BeforeClass annotation

File: src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            GraziConfig.state.enabledLanguages.add(Lang.RUSSIAN)
+            GraziLifecycle.publisher.reInit()
+        }
+
+        runHighlightTestForFile("ide/language/plain/ExampleRU.txt")
+
+        if (!isRussianEnabled) {
```
Comment (TanVD) [inline src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt]: AfterClass?

File: src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt [hunk 4]
```diff
@@ -0,0 +1,25 @@
+package tanvd.grazi.ide.language
+
+import tanvd.grazi.GraziConfig
+import tanvd.grazi.GraziTestBase
+import tanvd.grazi.ide.GraziLifecycle
+import tanvd.grazi.language.Lang
+
+
+class MultiLanguageTextSupportTest : GraziTestBase(true) {
```
Comment (TanVD) [inline src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt:10]: If it is actually `MultiLanguage text support` then you need to add English in text of test :)

File: src/main/kotlin/tanvd/grazi/language/LangTool.kt (hunk 1 above)
Comment (Fatalll) [reply src/main/kotlin/tanvd/grazi/language/LangTool.kt]: Done

File: src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt (hunk 2 above)
Comment (Fatalll) [reply src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt]: Move to setUp and tearDown

File: src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt (hunk 3 above)
Comment (Fatalll) [reply src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt]: Move to tearDown

File: src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt (hunk 4 above)
Comment (Fatalll) [reply src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt:10]: Add English text and now it's md file

**Pull request merged**